install/azure-installation-guide.md

```diff
@@ -0,0 +1,18 @@
+# Install Kata Containers on Azure
+
+Kata Containers on Azure use nested virtualization to provide an identical installation
+experience to Kata on your preferred Linux distribution. 
+
+This guide assumes you have an Azure account set up and tools to remote into your virtual
```

Comment:
Nit: "... to remotely login to your..." would be clearer I think.

Reply:
done